site/src/components/PrereqGraph.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <Network
+    ref="network"
+    :nodes="nodes"
+    :edges="edges"
+    :options="options"
+  ></Network>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from "vue-property-decorator";
+
+// @ts-expect-error: No types for this, unfortunately
+import * as VueVisJs from "vue-visjs";
+const Network = VueVisJs.Network;
+
+interface Node {
+  id: number;
+  label: string;
+  depth: number;
+  color: Color;
+  font: {
+    size: number;
+  };
+}
+
+interface Edge {
+  from: number;
+  to: number;
+}
+
+interface Graph {
+  nodes: Node[];
+  edges: Edge[];
+}
+
+interface Color {
+  background: string;
+  border: string;
+  highlight: {
+    background: string;
+    border: string;
+  };
+}
+
+@Component({
+  components: {
+    Network,
+  },
+  computed: {
+    options: function () {
```

Comment:
Is this function able to be moved into the class?

Reply:
Yes, fixed